readme.md

```diff
@@ -1,4 +1,4 @@
-# <img src="distribution/icon.png" width="45" align="left"> Refined GitHub
+# <img src="distribution/icon.png" width="45" align="left"> Standardised PR Process
```

Comment:
naming convention
variable type
------------------------------------------------
ISSUE CODE:
JS001 - Variable naming convention should be correct
JS002 - Define type for each variable